c2corg_ui/static/partials/contexthelp/xreport-increase_impact.html

```diff
@@ -0,0 +1,10 @@
+<p translate>If relevant, describe the factors which might have made the situation worse:</p>
+<ul>
+  <li><span translate>remoteness</span></li>
+  <li><span translate>level of commitment of the route</span></li>
+  <li><span translate>not being prepared for an incident</span></li>
+  <li><span translate>lack of experience of what to do in an incident/accident</span></li>
+  <li><span translate>lack of equipment</span></li>
+  <li><span translate>etc.</span></li>
```

Comment:
Do you really need to translate ``etc.`?
If not simply remove ``<span translate>``.